.github/workflows/test.yml

```diff
@@ -0,0 +1,135 @@
+name: Test
+
+on:
```

Comment:
Our config for main repo is this:

https://github.com/RDFLib/rdflib/blob/6aefe60ecd1f4c0b069919d898b3ed42a2d9cd03/.github/workflows/validate.yaml#L3-L7

```
on:
  push:
    branches: ["master"]
  pull_request:
  workflow_dispatch:
```

And it is maybe a bit more appropriate